grakn-engine/src/main/java/ai/grakn/engine/postprocessing/ConceptFixer.java

```diff
@@ -84,7 +84,6 @@ private static int performRetry(int retry){
         retry ++;
         double seed = 1.0 + (Math.random() * 5.0);
         double waitTime = (retry * 2.0)  + seed;
-        LOG.error(ErrorMessage.BACK_OFF_RETRY.getMessage(waitTime));
```

Comment:
I hastily removed it because it was "error" and printed all the time. We can add it back in as "info" or "debug"